backend/migrations/env.py

```diff
@@ -1,6 +1,21 @@
 from __future__ import with_statement
 from alembic import context
 from logging.config import fileConfig
+import re
+
+
+# DO NOT DELETE
+# import additional models here
+# they are not used directly in env.py, but these imports are important for alembic
+
```

Comment:
We don't know the root cause of alembic ignoring these models right?

Reply:
No, I couldn't find it. I found only a guide, where it was said that from multiple files you should explicitly import not only Base, but the models as well.
But honestly I couldn't understand, why exactly this models are unseen without explicit import.